api/pyproject.toml

```diff
@@ -32,7 +32,7 @@ dependencies = [
     "numpy",
     "pandas",
     "requests",
-    "openai==1.61.0",
+    "openai==1.69.0",
```

Comment:
any specific reason for this upgrade? I pinned the earlier version because the open ai upgrades were breaking the litellm version we were using. I was hoping to upgrade litellm and open ai together when I got a chance, but if you tested this change, and it works, it's great!